src/components/CollectionSummary/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { ArrowDropDown, ArrowDropUp } from '@mui/icons-material';
```

Comment:
We discussed we should use free FA icons instead of `@mui/icons-material`, right?